integration/build.gradle

```diff
@@ -0,0 +1,96 @@
+description = 'OpenCensus Java interop test client and server'
+
+apply plugin: 'idea'
+apply plugin: 'java'
+apply plugin: 'com.google.protobuf'
+
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+    }
+    dependencies {
+        // ASSUMES GRADLE 2.12 OR HIGHER. Use plugin version 0.7.5 with earlier
+        // gradle versions
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.3'
+    }
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+group = "io.opencensus"
+version = "0.13.0-SNAPSHOT" // CURRENT_OPENCENSUS_VERSION
+
+def opencensusVersion = "0.12.2" // LATEST_OPENCENSUS_RELEASE_VERSION
```

Comment:
should we make this a project property to allow overriding this with `-PopencensusVersion=...` on the command line (for local testing with a checked out version built into mavenLocal)?

Reply:
Make sense, I added a check here. gRPC version should also be configurable depending on the OpenCensus version we use.